noxfile.py

```diff
@@ -33,7 +33,7 @@
 LINT_PATHS = ["docs", "pandas_gbq", "tests", "noxfile.py", "setup.py"]
 
 DEFAULT_PYTHON_VERSION = "3.10"
-
```

Comment:
I'm not sure I understand why we're introducing this, since it's the same as the default?  Maybe explain this in the PR description?